go/host/l1/publisher.go

```diff
@@ -221,12 +228,65 @@ func (p *Publisher) PublishSecretResponse(secretResponse *common.ProducedSecretR
 	return nil
 }
 
-// ExtractObscuroRelevantTransactions will extract any transactions from the block that are relevant to obscuro
+// ExtractRelevantTenTransactions will extract any transactions from the block that are relevant to TEN
 // todo (#2495) we should monitor for relevant L1 events instead of scanning every transaction in the block
-func (p *Publisher) ExtractObscuroRelevantTransactions(block *types.Block) ([]*ethadapter.L1RespondSecretTx, []*ethadapter.L1RollupTx, []*ethadapter.L1SetImportantContractsTx) {
-	var secretRespTxs []*ethadapter.L1RespondSecretTx
-	var rollupTxs []*ethadapter.L1RollupTx
-	var contractAddressTxs []*ethadapter.L1SetImportantContractsTx
+func (p *Publisher) ExtractRelevantTenTransactions(block *types.Block, receipts types.Receipts) ([]*common.TxAndReceiptAndBlobs, []*ethadapter.L1RollupTx, []*ethadapter.L1SetImportantContractsTx) {
```

Comment:
maybe in a future PR, (when the rummaging stuff is also fixed), we should move this logic to a different class, because this is not really a "publishing" responsibility. 
Or a rename.

Reply:
agreed, I'll stick that in a ticket for monday